src/wp-includes/html-api/class-wp-html-processor.php

```diff
@@ -1116,7 +1116,7 @@ public function serialize(): ?string {
 		if ( WP_HTML_Tag_Processor::STATE_READY !== $this->parser_state ) {
 			wp_trigger_error(
 				__METHOD__,
-				"An HTML Processor which has already started processing cannot serialize it's contents. Serialize immediately after creating the instance.",
+				'An HTML Processor which has already started processing cannot serialize its contents. Serialize immediately after creating the instance.',
```

Comment:
[Single quotes are supposed to be used here, the linter would flag this.](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#single-and-double-quotes) It was allowed before this change because the string included a single quote. This is one of the conditions that allows doubled-quoted strings:

- `"not OK"`
- `"it's ok"`

I'm unaware of specific guidelines around whether this should be localized or not. I've reviewed the handbook and nothing specific is mentioned. These strings that are more developer-facing are a gray area.

Reply:
I saw translated strings passed, as well as untranslated in the codebase. It's a developer-facing interface, so given the code is in English, the server request can be served with multiple locales enforced, it's probably best to keep it untranslated as long it doesn't get exposed in the UI.